c/loop-industry-pattern/aiob_1_true-unreach-call.c

```diff
@@ -122,8 +122,12 @@ INT16* Id_MCDC_87;
 } ;
 typedef long  INT64;
 typedef float  REAL32;
-extern UINT32 Id_MCDC_92(UINT16 Id_MCDC_88, UINT8 Id_MCDC_89, UINT32 Id_MCDC_90, void * Id_MCDC_91);
-extern UINT32 Id_MCDC_92(UINT16 Id_MCDC_88, UINT8 Id_MCDC_89, UINT32 Id_MCDC_93, void * Id_MCDC_91);
+UINT32 Id_MCDC_92(UINT16 Id_MCDC_88, UINT8 Id_MCDC_89, UINT32 Id_MCDC_90, void * Id_MCDC_91);
+UINT32 Id_MCDC_92(UINT16 Id_MCDC_88, UINT8 Id_MCDC_89, UINT32 Id_MCDC_93, void * Id_MCDC_91)
+{
+	UINT32 x;
```

Comment:
Even though it seems that this variable is not being used anywhere, I would still prefer if it would be initialized with a call to a nondet function. Just to be sure we are avoiding undefined behaviors.

Reply:
This holds for all 3 benchmarks.